pkg/prowgen/prowgen_owners.go

```diff
@@ -23,12 +24,24 @@ func GenerateOwners(ctx context.Context, configs []*Config) error {
 		config := config
```

Comment:
I think we should always update midstream `main` with owners as well. But that's unrelated to this fix.

Reply:
(if I understood you correctly)